Caw.h

```diff
@@ -0,0 +1,20 @@
+#include <string>
+#include <gflags/gflags.h>
+
+DECLARE_string(user);
```

Comment:
You probably don't need access to these in this file.

Reply:
Removed unecessary includes